config/mock-api-data.js

```diff
@@ -11,6 +11,26 @@ function * range (max)
     return max;
 }
 
+function setTagName () {
+    
+    const tagsArray = [
+        'florist',
+        'grocery',
+        'pharmacy',
+        'newsstand',
+        'butcher',
+        'bakery',
+        'cafe',
+        'coffeshop',
+        'clothes',
+        'furnishings'
```

Comment:
Missing trailing comma, trailing whitespace, OTBS brackets... Please consider checking linter messages and using editorconfig plugins ;)

Reply:
Fixed in 1b410e3